pkg/imageset/updateinstaller_test.go

```diff
@@ -28,64 +29,69 @@ import (
 const (
 	testInstallerImage = "registry.io/test-installer-image:latest"
 	testCLIImage       = "registry.io/test-cli-image:latest"
+	testReleaseVersion = "v0.0.0-test-version"
 )
 
 func TestUpdateInstallerImageCommand(t *testing.T) {
 	apis.AddToScheme(scheme.Scheme)
 
 	tests := []struct {
-		name                      string
+		name    string
+		images  map[string]string
+		version string
```

Comment:
I was tempted to request an e2e test for this, but as implemented the job will fail if something is wrong, so in theory we're already getting coverage that this code works. I'm not sure we'd want to assert specific versions in the e2e test anyhow.

Reply:
The test was testing there are on errors and not the actual values and as you said it already covers testing the release version code path..